woocommerce/woocommerce-ios
# [Product Bundles] Release read-only support for product bundles

## Description

This enables the product bundles feature flag, to release read-only support for product bundles.

⚠️ This should not be merged until #9200 is merged.

## Testing instructions

Please follow the mini test plan (internal ref: pe5pgL-2tL-p2):

### Prerequisites

1. Install and activate the Product Bundles extension on your store.
2. Create one or more products with the “Product Bundle” product type in wp-admin. Some ideas:
   - Create a product bundle but don’t add any bundled products to it.
   - Create a product bundle with one or more bundled products.
   - Change the inventory settings on the product bundle to manage the stock quantity, or just use the stock status.
   - Change the inventory settings of the bundled products (in their product details, not from within the product bundle details) so that some of the bundled products are out of stock.

### To test

- Test that all of your products, including product bundles, load in the product list on the Products tab.
- Test that the correct stock status or stock quantity is displayed for product bundles in the product list on the Products tab (same as the product list in wp-admin).
- Test that you can open the product details for a product bundle, with read-only support for bundled products, price, and inventory.
- Test that a “Bundled products” row appears in product details for product bundles, and it shows the correct number of bundled products.
- Test that tapping “Bundled products” opens a list with each bundled product’s image, name, and stock status (in stock, out of stock, or on backorder).
- Test that you can make changes to the editable details on your product bundle (e.g. title, description) and the changes are saved/published correctly.

## Screenshots


https://user-images.githubusercontent.com/8658164/225636473-d0a2d0f8-04b0-47ce-a8f4-9c98251871aa.mp4



---
